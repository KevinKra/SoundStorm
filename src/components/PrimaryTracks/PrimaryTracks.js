import React from "react";
import "./PrimaryTracks.scss";
import TrackList from "../../containers/TrackList/TrackList";

export const PrimaryTracks = () => {
  return (
    <section className="PrimaryTracks">
      <TrackList
        genre="hiphop"
        count={6}
        listTitle="Featured HipHop"
        width="32%"
      />
      <TrackList genre="rock" count={6} listTitle="Featured Rock" width="32%" />
      <TrackList
        genre="country"
        count={6}
        listTitle="Featured Country"
        width="32%"
      />
    </section>
  );
};
