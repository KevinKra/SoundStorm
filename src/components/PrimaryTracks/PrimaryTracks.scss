.PrimaryTracks {
  display: flex;
  justify-content: space-between;
  // flex-direction: column;
  // border: 2px solid blue;
  height: 70vh;
  margin-bottom: 1rem;
  width: 100%;
  .TrackList {
    // border: 1px solid green;
    height: 100%;
    width: 50%;
  }
  .TrackDetails {
    position: relative;
    transform: translateX(0%);
    border: 1px solid blue;
    width: 45%;
    height: 90%;
  }
  .TrackCard {
    margin-top: 0.5rem;
    border: 1px solid red;
    width: 100%;
    height: 13%;
  }
}
