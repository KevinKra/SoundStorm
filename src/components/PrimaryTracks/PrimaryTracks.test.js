import React from "react";
import { PrimaryTracks } from "./PrimaryTracks";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("<PrimaryTracks />", () => {
  let wrapper = shallow(<PrimaryTracks />);
  it("should match the snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
