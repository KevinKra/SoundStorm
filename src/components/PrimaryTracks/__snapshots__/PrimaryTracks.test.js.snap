// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrimaryTracks /> should match the snapshot 1`] = `
<section
  className="PrimaryTracks"
>
  <Connect(TrackList)
    count={6}
    genre="hiphop"
    listTitle="Featured HipHop"
    width="32%"
  />
  <Connect(TrackList)
    count={6}
    genre="rock"
    listTitle="Featured Rock"
    width="32%"
  />
  <Connect(TrackList)
    count={6}
    genre="country"
    listTitle="Featured country"
    width="32%"
  />
</section>
`;
